src/client/localplayer.cpp

```diff
@@ -727,10 +735,13 @@ v3f LocalPlayer::getEyeOffset() const
 
 // 3D acceleration
 void LocalPlayer::accelerate(const v3f &target_speed, const f32 max_increase_H,
-		const f32 max_increase_V, const bool use_pitch)
+		const f32 max_increase_V, bool use_pitch)
 {
 	const f32 yaw = getYaw();
 	const f32 pitch = getPitch();
+	if (use_pitch && touching_ground && pitch > 0.0f)
+		// Prevent player from slowing down when looking at the ground in pitch mode
```

Comment:
Even though this is technically correct, please add brackets around this if-block.